brainscore/benchmarks/__init__.py

```diff
@@ -73,12 +73,13 @@ class BenchmarkBase(Benchmark):
     """
     Helper class for implementing standard functions of the `Benchmark` interface.
     """
-    def __init__(self, identifier, ceiling_func, version, parent=None, paper_link=None):
+    def __init__(self, identifier, ceiling_func, version, parent=None, paper_link=None, bibtex=None):
         self._identifier = identifier
         self._ceiling_func = ceiling_func
         self._version = version
         self.parent = parent
         self.paper_link = paper_link
+        self.bibtex = bibtex
```

Comment:
maybe we should also add this as a property to the `Benchmark` interface (which `BenchmarkBase` implements), i.e.:
```
    @property
    def bibtex(self):
        """
        bibtex string to build the reference.
        Should include an `url` to build a proper link.
        """
        raise NotImplementedError()
```